event_webhook_types_test.go

```diff
@@ -101,6 +101,10 @@ func TestCommitCommentEventUnmarshal(t *testing.T) {
 		t.Errorf("NoteableType is %v, want %v", event.ObjectAttributes.NoteableType, "Commit")
 	}
 
+	if event.ObjectAttributes.Action != "create" { //nolint:goconst
```

Comment:
`//nolint` exists because if there are >2 usages of the `"create"` string, the linter will complain about
```
string `create` has 3 occurrences, but such constant `FileCreate` already exists (goconst)
```
Using `FileCreate` is the wrong intent for me, please let me know how you want that handled. I could also add a description why that is in the code

Reply:
Maybe create a const for it? Something like:

```go
type CommentEventAction string

const (
  CommentEventActionCreate CommentEventAction = "create"
  CommentEventActionUpdate CommentEventAction = "update"
)
```

Reply:
Thanks for the idea and snippet! I also used the action struct in the event struct. Let me know if this should live somewhere else, I used `StateID` as a reference